build.py

```diff
@@ -95,6 +105,33 @@
 wheels_dir = os.path.join(root_dir, "target", "wheels")
 vscode_src = os.path.join(root_dir, "vscode")
 jupyterlab_src = os.path.join(root_dir, "jupyterlab")
+package_name = "qsharp-nightly"
+
+if os.environ.get("QSHARP_PACKAGE_SUFFIX") is not None:
+    suffix = os.environ["QSHARP_PACKAGE_SUFFIX"]
+    if suffix == "stable":
+        package_name = "qsharp"
+    else:
+        package_name = f"qsharp-{suffix}"
+
+    update_package_name_args = [
+        sys.executable,  # use the current python, we aren't installing anything
+        "update_package_suffix.py",
+        suffix,
+    ]
+    print(f"Updating package name for {package_name}")
+    subprocess.run(update_package_name_args, check=True, text=True, cwd=root_dir)
+
+if os.environ.get("QSHARP_PACKAGE_VERSION") is not None:
+    version = os.environ["QSHARP_PACKAGE_VERSION"]
+
+    update_package_version_args = [
+        sys.executable,  # use the current python, we aren't installing anything
+        "update_package_version.py",
+        version,
+    ]
+    print(f"Updating packages version to {version}")
+    subprocess.run(update_package_version_args, check=True, text=True, cwd=root_dir)
```

Comment:
I prefer the way we call out to `prereqs.py` above, where it's just a normal function import. `prereqs.py` is also runnable as a standalone script (the `if __name__ == "__main__":` check below). 

Since these are both helper scripts that get called from `build.py`, it'd be nice if we followed the same pattern in both. I like symmetry.

Reply:
Also, being that they are largely doing similar work on roughtly the same set of files, would seem optimial do have one script for updating name and version, rather than two that duplicate a lot of code/work.